src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/util/BlackDuckResponseCache.java

```diff
@@ -105,14 +106,14 @@ public String getProjectComponentQueryLink(String projectLink, String componentN
         return Optional.empty();
     }
 
-    public String getSeverity(String vulnerabilityUrl) {
-        String severity = "UNKNOWN";
+    public ProjectVersionVulnerableBomComponentsItemsVulnerabilityWithRemediationSeverityType getSeverity(String vulnerabilityUrl) {
```

Comment:
Why did we change the return type from String to ProjectVersionVulnerableBomComponentsItemsVulnerabilityWithRemediationSeverityType?  Instead of the users of the method calling the name() method on the response why dont we do that at the end of this method?

Reply:
good question, I'll change that